### PR TITLE
Preview text

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@lukehoban @bermudezmt for help with wordsmithing here.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,20 @@
 [![Build Status](https://travis-ci.com/pulumi/pulumi-cloud.svg?token=eHg7Zp5zdDDJfTjY8ejq&branch=master)](https://travis-ci.com/pulumi/pulumi-cloud)
 
-# Pulumi Cloud Framework
+# Pulumi Cloud Framework (Preview)
 
 Pulumi's multi-cloud framework for building modern container and serverless cloud applications.
```
Comment (bermudezmt) [inline README.md]:
```suggestion
Pulumi Cloud Framework is our multi-cloud framework for building modern container and serverless cloud applications.
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
 
+This is a preview library demonstrating how to build Pulumi [component
+resources](https://www.pulumi.com/docs/reference/component-tutorial/) that can abstract over and
```
Comment (bermudezmt) [inline README.md]:
```suggestion
(https://www.pulumi.com/docs/reference/component-tutorial/) that can abstract over and
```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,20 @@
 [![Build Status](https://travis-ci.com/pulumi/pulumi-cloud.svg?token=eHg7Zp5zdDDJfTjY8ejq&branch=master)](https://travis-ci.com/pulumi/pulumi-cloud)
 
-# Pulumi Cloud Framework
+# Pulumi Cloud Framework (Preview)
 
 Pulumi's multi-cloud framework for building modern container and serverless cloud applications.
 
+This is a preview library demonstrating how to build Pulumi [component
```
Comment (bermudezmt) [inline README.md]:
```suggestion
This is a preview library demonstrating how to build Pulumi [components]
```

File: README.md [hunk 4]
```diff
@@ -1,9 +1,20 @@
...
+resources](https://www.pulumi.com/docs/reference/component-tutorial/) that can abstract over and
+target multiple clouds.
+
+For developers targeting a single cloud platform like AWS, Azure or GCP, we recommend using the
+[@pulumi/aws](https://github.com/pulumi/pulumi-aws),
+[@pulumi/azure](https://github.com/pulumi/pulumi-azure) and
+[@pulumi/gcp](https://github.com/pulumi/pulumi-gcp) packages respectively.  These packages give full
```
Comment (bermudezmt) [inline README.md]: If we go with my suggestion, I'll just try to make it consistent here.

> A platform-specific package gives you full access to the breadth of its capabilities and comes with many abstractions to make development easier.

**Pull request merged**